app.js

```diff
@@ -0,0 +1,80 @@
+const bookContainer = document.querySelector('.books-container');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const addBtn = document.querySelector('.add-btn');
+let i = 0;
+
+// const books = [];
+let myLibrary = [];
```

Comment:
Kindly make sure that a new book is added to the collection (in your case the `myLibrary` array). Great job saving the books in the localStorage, however, every new book must also be saved in both the local array and localStorage simultaneously.

- Currently, when I check your local array after creating a new book, it's empty.
- You can achieve this by pushing the new book object to the local array first, then update the local storage.